docs/docs/api-reference.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Gatsby API Reference
+title: Référence de l'API Gatsby
 overview: true
 ---
 
-Learn more about Gatsby API methods and options, common files, and customizations.
+En savoir plus sur les méthodes et options de l'API Gatsby, les fichiers communs et les personnalisations.
```

Comment:
As it is not an actionnable component but only a standalone sentence, I think the following change make more sense:
 
```suggestion
Apprenez-en plus sur les méthodes et options de l'API Gatsby, les fichiers communs et les personnalisations.
```